frontend/mr-admin-flate/src/AutentisertTiltaksansvarligApp.tsx

```diff
@@ -53,6 +54,15 @@ export default function AutentisertTiltaksansvarligApp({ ansatt }: Props) {
         }
         errorElement={<ErrorPage />}
       />
+      <Route
+        path="opprett"
```

Comment:
Tror det er lurt å være mer eksplisitt å kalle den feks `opprett-tiltaksgjennomforing`. Grunnen er at vi vet ikke enda om det kommer flere operasjoner for "å opprette noe..." i fremtiden